nubis/puppet/init.pp

```diff
@@ -9,3 +9,28 @@
 import 'fluentd.pp'
 import 'nubis_configuration.pp'
 #include nubis_configuration
+
+exec { "apt-get update":
```

Comment:
This is an elegant little bit of puppet, but might it not better belong as a generic provisionning step in nubis-builder, maybe ?